package.json

```diff
@@ -78,6 +85,7 @@
     "ncp": "^2.0.0",
     "object-assign": "^4.1.1",
     "pacote": "^10.3.0",
+    "parse-filepath": "^1.0.2",
```

Comment:
nit: These `devDependencies` should be moved to `packages/react-devtools-extensions/package.json` since it's the only package that uses them.